modules/components/pages/processors/gcp_vertex_ai_chat.adoc

```diff
@@ -0,0 +1,207 @@
+= gcp_vertex_ai_chat
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+Generates responses to messages in a chat conversation, using the Vertex AI API.
+
+Introduced in version 4.34.0.
+
+
+[tabs]
+======
+Common::
++
+--
+
+```yml
+# Common configuration fields, showing default values
+label: ""
+gcp_vertex_ai_chat:
+  project: "" # No default (required)
+  credentials_json: "" # No default (optional)
+  location: us-central1 # No default (optional)
+  model: gemini-1.5-pro-001 # No default (required)
+  prompt: "" # No default (optional)
+  temperature: 0 # No default (optional)
+  max_tokens: 0 # No default (optional)
+  response_format: text
+```
+
+--
+Advanced::
++
+--
+
+```yml
+# All configuration fields, showing default values
+label: ""
+gcp_vertex_ai_chat:
+  project: "" # No default (required)
+  credentials_json: "" # No default (optional)
+  location: us-central1 # No default (optional)
+  model: gemini-1.5-pro-001 # No default (required)
+  prompt: "" # No default (optional)
+  system_prompt: "" # No default (optional)
+  temperature: 0 # No default (optional)
+  max_tokens: 0 # No default (optional)
+  response_format: text
+  top_p: 0 # No default (optional)
+  top_k: 0 # No default (optional)
+  stop: [] # No default (optional)
+  presence_penalty: 0 # No default (optional)
+  frequency_penalty: 0 # No default (optional)
+```
+
+--
+======
+
+This processor sends prompts to your chosen large language model (LLM) and generates text from the responses, using the Vertex AI API.
+
+For more information, see the https://cloud.google.com/vertex-ai/docs[Vertex AI documentation^].
+
+== Fields
+
+=== `project`
```

Comment:
@rockwotj. I have made minor amends to the field descriptions. Please could you check that all is correct?

Reply:
Looks good! I will update the source with these